src/library/Form/FormField.js

```diff
@@ -37,20 +37,23 @@ type Props = {
 
 const REGEX_GROUP = /(Checkbox|Radio|Group)/i;
 
+// [1] This deviates from h6 because it must look right when used both within
+//     and without a FormFieldset
 export const componentTheme = (baseTheme: Object) => ({
-  FormFieldCaption_color_text: baseTheme.color_gray_80,
+  FormFieldCaption_color_text: baseTheme.color_mouse,
   FormFieldCaption_fontSize: baseTheme.fontSize_mouse,
   FormFieldCaption_marginTop: baseTheme.space_stack_xxs,
   FormFieldCaption_marginTop_isGroup: baseTheme.space_stack_sm,
 
-  FormFieldLabel_color_text: baseTheme.color_gray_80,
-  FormFieldLabel_fontSize: baseTheme.fontSize_ui,
-  FormFieldLabel_fontWeight: baseTheme.fontWeight_semiBold,
+  FormFieldLabel_color_text: baseTheme.color_h6,
+  FormFieldLabel_fontSize: baseTheme.fontSize_h6,
+  FormFieldLabel_fontWeight: baseTheme.fontWeight_semiBold, // [1]
```

Comment:
Did you mean to leave these `1` comments in (one in MenuGroupTitle as well)? Do they reference something?

Reply:
Yep. There's a comment block on line 40, just before defining the `const`. Same for MenuGroupTitle.